R/tbl-lazy.R

```diff
@@ -114,12 +109,12 @@ names.tbl_lazy <- function(x) {
       i = "Did you mean {.fn colnames}?"
     ))
   }
-  NextMethod("names")
+  names(unclass(x))
```

Comment:
Without this, `names.tbl_lazy` gets called twice. Presumably some `NextMethod()` weirdness.